tests/src/Functional/ContactsDashboardTest.php

```diff
@@ -32,6 +32,7 @@ protected function setUp() {
     $this->adminUser->save();
 
     \Drupal::service('theme_installer')->install(['contacts_theme']);
```

Comment:
With it doing the install as part of `hook_requirements()` (or soon to be `hook_install()`), do we need to do it separately here? And, if not, can `contacts` just go in `self::$modules`?